### PR TITLE
bundle edit_line converge fixups, converge_prepend addition/master

### PR DESCRIPTION
**Pull request by nickanderson**
Superceeds https://github.com/cfengine/masterfiles/pull/1853

### CODE REVIEW

**Comment by tzz** [comment]
YES

**Comment by nickanderson** [comment]
@cf-bottom can you please have Jenkins build this since Travis is so epically slow.

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/5892/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/5892/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-5892/

File: tests/acceptance/lib/files/edit_line_converge_prepend.cf [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+     "$(G.testfile).expected"
+      copy_from => local_cp("$(this.promise_filename).finish");
+}
+
+#######################################################
+
+bundle agent test
+{
+  meta:
+      "description"
```
Comment (nickanderson) [inline tests/acceptance/lib/files/edit_line_converge_prepend.cf:26]: add promisee -> { "CFE-3483" }

File: lib/files.cf [hunk 2]
```diff
@@ -1071,11 +1071,30 @@ bundle edit_line converge(marker, lines)
...
   insert_lines:
       "$(lines)" comment => "Insert the given lines";
 }
 
+bundle edit_line converge_prepend(marker, lines)
+# @brief Converge `lines` marked with `marker` to start of content
+#
+# Any content marked with `marker` is removed, then `lines` are
+# inserted at *start* of content.  Every `line` should contain `marker`.
+#
```
Comment (nickanderson) [inline lib/files.cf:1084]: missing example, missing history

**Comment by nickanderson** [comment]
@cf-bottom please run through Jenkins, again.

**Comment by nickanderson** [comment]
Alright, @cf-bottom,  please run this through Jenkins one more time.

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/5894/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/5894/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-5894/

**Comment by nickanderson** [comment]
Now that's the green I was looking for.

**Pull request merged**